src/Database/LSMTree/Common.hs

```diff
@@ -107,7 +107,8 @@ openSession ::
   -> m (Session m)
 openSession = undefined
 
--- | Close the table session. 'closeSession' is idempotent.
+-- | Close the table session. 'closeSession' is idempotent. All operations on
+-- the session and tables in it will throw an exception.
```

Comment:
```suggestion
-- | Close the table session. 'closeSession' is idempotent. All subsequent
-- operations on the session or the tables within it will throw an exception.
```